Djikstra/Djikstra/Graph.cpp

```
@@ -0,0 +1,181 @@
+#include <vector>
+#include <queue>
+#include <deque>
+#include <map>
+#include <fstream>
+#include <string>
+
+
+#include "Graph.h"
+
+
+namespace Graph
+{
+    class Comparator {
+    public:
+        bool operator()(const Vertex& v1, const Vertex& v2)
+        {
+            return v1.distance > v2.distance;
+        }
+    };
+
+
+
+    void Graph::load_graph() {
+        std::string name1, name2;
+        float dist = 0;
+
+        std::fstream file_data;
+        file_data.open("distances.txt", std::ios::in);
+
+        if (file_data.good() == false)
+        {
+            std::cout << "Error with oppening";
```

You should raise a proper exception.